README.rst

```diff
@@ -73,6 +73,19 @@ If you wish to set a threshold for iviewer that is *lower* than for the server:
 NB: Z-projection is not supported for tiled images in OMERO
 (Images larger than 2048 * 2048 pixels per plane are tiled in iviewer).
 
+OMERO uses Spectrum Color Picker for selecting ROI colors. 
+The roi_color_palette option allows you to specify a grid of colors for users to choose for ROIs.
+Define rows with brackets, and use commas to seperate values. 
+You can only view the first item of each row without enabling show_palette_only.
```

Comment:
Hopefully this is a bit clearer suggestion: "By default, only the first color of each row is shown. A full grid is shown when the default color picker is hidden (see below)".